src/tests/functional/plugin/conformance/test_runner/op_conformance_runner/src/read_ir/read_ir.cpp

```diff
@@ -238,36 +239,21 @@ void ReadIRTest::SetUp() {
         if (param->get_partial_shape().is_static()) {
             inputShapes.push_back(InputShape{{}, {param->get_shape()}});
         } else {
-            std::vector<ov::Shape> staticShapes = { param->get_partial_shape().get_min_shape(),
-                                                    param->get_partial_shape().get_min_shape(),
-                                                    param->get_partial_shape().get_max_shape() };
-            ov::Shape midShape;
-            for (const auto s : param->get_partial_shape()) {
-                int dimValue = 1;
-                if (s.is_dynamic()) {
-                    size_t range = s.get_max_length() - s.get_min_length();
-                    if (range > std::numeric_limits<char>::max()) {
-                        ov::test::utils::fill_data_random(&range, 1, std::numeric_limits<char>::max(), s.get_min_length(), 1);
-                    }
-                    ov::test::utils::fill_data_random(&dimValue, 1, range, s.get_min_length(), 1);
-                } else {
-                    dimValue = s.get_length();
-                }
-                midShape.push_back(dimValue);
-            }
-            staticShapes[1] = midShape;
 
-            // Shape validation to avoid large values
+            const auto paramConsumer = *param->get_output_target_inputs(0).begin();
```

Comment:
I've been thinking about this case. However, I still have no idea how to resolve this situation.

Perhaps this can be ignored in the test conditions.
If each test verifies a specific operation, then the appropriate data for the operation under test can always be placed on the first port?

What do you think about this?

Reply:
What do you think about to define limitations  per output node and aggregate it in input generation?

Honestly, it should be fixed anyway, but we can split this case to separated PR. Thanks for challenging!

FYI I have approved https://github.com/openvinotoolkit/openvino/pull/24162 recently. So you need to rebase your branch to latest master in merge case!

Many thanks! So wonderful to review such PRs!